tests/src/core/texture/Texture2D.test.ts

```
@@ -111,6 +111,36 @@ describe("Texture2D", () => {
+      expect(buffer[3]).to.eq(4);
+      expect(buffer[4]).to.eq(7);
+      expect(buffer[5]).to.eq(4);
+      expect(buffer[6]).to.eq(3);
+      expect(buffer[7]).to.eq(3);
+    });
+
+    it.only("读取 R8 成功", () => {
+      const texture = new Texture2D(engine, width, height, TextureFormat.R8);
+      const buffer = new Uint8Array(4);
+
+      texture.setPixelBuffer(new Uint8Array([1, 6, 8, 4]), 0, 5, 0, 2, 2);
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)